pegasus/sites.v3/codeprojects.org/styles/050-theme.css

```diff
@@ -1,43 +1,54 @@
 body {
-  padding-bottom: 28px;
+  padding-bottom: 28px !important;
 }
 
-#pagefooter {
+footer#codeprojects_pagefooter {
+  all: initial; /* blocking inheritance for all properties */
```

Comment:
This tag does seem handy.  I guess the biggest risk is that we have a bifurcation between browsers that support it and those that don't (i.e. it'll solve clashes in some browsers but not others) but I think I'm convinced that it is useful when it works and no worse than not having it when it doesn't work :)